portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -97,6 +104,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(comments));
  }
+
+ private int getLimit(HttpServletRequest request){
+     String limitString = request.getParameter("limit");
+
+     int limit;
+     try{
+      limit = Integer.parseInt(limitString);   
+     } catch(NumberFormatException e) {
+         System.err.println("Could not convert to int: " + limitString);
+         return -1;
+     }
+     if(limit < 0){
+        System.err.println("Player choice is out of range: " + limitString);
+        return -1;
+     }
+     return limit;
+  }
   private Comment entityToComment(Entity entity){
```

Comment:
we should have an extra new line in between function definitions, just for added clarity.

Reply:
fixed!